phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -1507,6 +1508,16 @@ public MutationState close(CloseStatement statement) throws SQLException {
         return new MutationState(0, 0, connection);
     }
 
+    /**
+     * Supprort long viewIndexId only if client has explicitly set
+     * the QueryServices.LONG_VIEW_INDEX_ENABLED_ATTRIB connection property to 'true'.
+     * @return
+     */
+    private PDataType getViewIndexDataType() throws SQLException {
+        boolean supportsLongViewIndexId = "true".equalsIgnoreCase(connection.getClientInfo(LONG_VIEW_INDEX_ENABLED_ATTRIB));
```

Comment:
Can you please add a default value for ```LONG_VIEW_INDEX_ENABLED_ATTRIB``` to ```QueryServicesOptions``` and set it to false and look it up similar to :

```
boolean asyncIndexBuildEnabled = connection.getQueryServices().getProps().getBoolean(
                QueryServices.INDEX_ASYNC_BUILD_ENABLED,
                QueryServicesOptions.DEFAULT_INDEX_ASYNC_BUILD_ENABLED);
```

Reply:
Done